sdk/textanalytics/Azure.AI.TextAnalytics/src/NamedEntitySubType.cs

```diff
@@ -0,0 +1,152 @@
+        /// </summary>
+        public static readonly NamedEntitySubType Temperature = new NamedEntitySubType("Temperature");
+
+        private readonly string _value;
+
+        private NamedEntitySubType(string type)
+        {
+            Argument.AssertNotNull(type, nameof(type));
+            _value = type;
+        }
+
+        /// <summary>
+        /// Defines implicit conversion from string to NamedEntitySubType.
+        /// </summary>
+        /// <param name="type">string to convert.</param>
+        /// <returns>The string as a NamedEntitySubType.</returns>
+        public static implicit operator NamedEntitySubType(string type) => type != null ? new NamedEntitySubType(type) : default;
+
+        /// <summary>
+        /// Defines explicit conversion from NamedEntitySubType to string.
+        /// </summary>
+        /// <param name="type">NamedEntitySubType to convert.</param>
+        /// <returns>The NamedEntitySubType as a string.</returns>
+        public static explicit operator string(NamedEntitySubType type) => type._value;
+
+        /// <summary>
+        /// Compares two NamedEntitySubType values for equality.
+        /// </summary>
+        /// <param name="lhs">The first NamedEntitySubType to compare.</param>
+        /// <param name="rhs">The second NamedEntitySubType to compare.</param>
+        /// <returns>true if the NamedEntitySubType objects are equal or are both null; false otherwise.</returns>
+        public static bool operator ==(NamedEntitySubType lhs, NamedEntitySubType rhs) => Equals(lhs, rhs);
+
+        /// <summary>
+        /// Compares two NamedEntitySubType values for inequality.
+        /// </summary>
+        /// <param name="lhs">The first NamedEntitySubType to compare.</param>
+        /// <param name="rhs">The second NamedEntitySubType to compare.</param>
+        /// <returns>true if the NamedEntitySubType objects are not equal; false otherwise.</returns>
+        public static bool operator !=(NamedEntitySubType lhs, NamedEntitySubType rhs) => !Equals(lhs, rhs);
+
+        /// <summary>
+        /// Compares the NamedEntitySubType for equality with another NamedEntitySubType.
+        /// </summary>
+        /// <param name="other">The NamedEntitySubType with which to compare.</param>
+        /// <returns><c>true</c> if the NamedEntitySubType objects are equal; otherwise, <c>false</c>.</returns>
+        public bool Equals(NamedEntitySubType other) => _value == other._value;
+
+        /// <summary>
+        /// Determines whether the specified object is equal to the current object.
+        /// </summary>
+        /// <param name="obj">The object to compare with the current object.</param>
+        /// <returns><c>true</c> if the specified object is equal to the current object; otherwise, <c>false</c>.</returns>
+        public override bool Equals(object obj) => obj is NamedEntitySubType type && Equals(type);
```

Comment:
Per [guidelines](https://azure.github.io/azure-sdk/dotnet_implementation.html#dotnet-enums), we should include:
`[EditorBrowsable(EditorBrowsableState.Never)]`

Reply:
Fixed.